app_dart/dev/branches.txt

```diff
@@ -1,4 +1 @@
-flutter-2.2-candidate.10
-flutter-2.4-candidate.4
-experimental_skshaper
```

Comment:
skshaper may still be used, do you know if there are new commits to that branch?

Reply:
Latest commit is 2 weeks old - https://github.com/flutter/flutter/tree/experimental_skshaper and https://github.com/flutter/engine/tree/experimental_skshaper

\cc @jason-simmons 

Generally, these branches are broken and aren't showing anything regardless through Cocoon. Now that ci.yaml controls where we run checks on a branched PR, this file should no longer be necessary.

Reply:
As another point, I talked to @christopherfujino offline and we discussed replacing this with a request for https://cs.opensource.google/flutter/infra/+/main:config/lib/release_branches/branches.json instead